internal/runners/languages/update.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/blang/semver"
+	goversion "github.com/hashicorp/go-version"
```

Comment:
This change was necessary, as we have a Python version on the Platform that is not semantic, namely `2.7.18.1`.